src/index.ts

```diff
@@ -830,11 +831,25 @@ export type PreparedFn<T extends Dictionary<ParameterAlias>, U> = (
 	passthrough?: Params['passthrough'],
 ) => U;
 
+export type RetryParameters = {
+	canRetry?: (err: any) => boolean;
+	logger?: (
```

Comment:
Maybe calling this `onRetry`/onError` would make more sense since it's only called on an error/retry